src/sbt-test/sbteclipse/03-sbt-dependency-withsources/build.sbt

```diff
@@ -17,13 +17,16 @@ libraryDependencies ++= Seq(
 
 TaskKey[Unit]("verify-classpath-xml") := {
   val dir = baseDirectory.value
+  val repo = "/.cache/coursier/v1/https/repo1.maven.org/maven2"
```

Comment:
will this work on Macs?
my coursier cache lives in `~/Library/Caches/Coursier/v1`

Reply:
This is only really relevant while building and testing the plug-in prior to its release. of course if you are doing just that it will fail.

Reply:
yah I figured it might be an issue because im on a Mac M1 and wanted to help diagnose the failing test